scripts/ci/post-build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
You can replace this with either `--base-url` (I think) from the CLI

Reply:
Nice one! I also forgot to build in prod